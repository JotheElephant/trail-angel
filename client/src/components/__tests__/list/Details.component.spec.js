'use strict';

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Details from '../../list/Details.component';
import homeData from '../../../../__tests__/fixtures/home-data';

test('renders correctly', () => {
  const data = homeData[0];
  const tree = renderer.create(
    <Details  favorites={[]}
              {...data}
    />
  );
  expect(tree).toMatchSnapshot();
});