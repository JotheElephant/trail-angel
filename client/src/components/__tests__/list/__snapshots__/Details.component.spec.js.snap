exports[`test renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "column",
      }
    }>
    <View
      onLayout={[Function]}
      size={40}
      style={
        Object {
          "flex": 40,
          "flexDirection": "row",
        }
      }>
      <View>
        <Image
          source={
            Object {
              "uri": "https://maps.googleapis.com/maps/api/place/photo?maxwidth=400&photoreference=CoQBdwAAAHAO1waHOwzcT8DCkFgK1BeyTIYAbo3kgKCax8kW3OrcnpN5OmZ6D9M7MzIfQ_PBKop-GFr5u6Pks-5ohsp2LQKmaTOyhh11ctQ_42uCLOTOXMZZri4Khr9VtlVgYuqYcEhiOlq374JRs821q1LCSoEKAJSQjS9rkMV9Q_GkBLQKEhCDKC4MuQY7vjnc0DC5wh6rGhRzR8J7SGGrmS9wXxXN1yavkwOXPA&key=AIzaSyDB5zzhpIGeUa2tU4vsqkUAVH0a5q-i_As",
            }
          }
          style={
            Object {
              "height": -32,
              "marginTop": 32,
              "width": NaN,
            }
          } />
      </View>
    </View>
    <View
      size={60}
      style={
        Object {
          "flex": 60,
          "flexDirection": "row",
        }
      }>
      <View
        size={60}
        style={
          Object {
            "flex": 60,
            "flexDirection": "column",
          }
        }>
        <ScrollView
          automaticallyAdjustContentInsets={false}
          dataSource={
            ListViewDataSource {
              "items": 5,
            }
          }
          initialListSize={10}
          onContentSizeChange={[Function]}
          onEndReachedThreshold={1000}
          onKeyboardDidHide={undefined}
          onKeyboardDidShow={undefined}
          onKeyboardWillHide={undefined}
          onKeyboardWillShow={undefined}
          onLayout={[Function]}
          onScroll={[Function]}
          pageSize={1}
          removeClippedSubviews={true}
          renderRow={[Function]}
          renderSeparator={[Function]}
          scrollEventThrottle={50}
          scrollRenderAheadDistance={1000}
          stickyHeaderIndices={Array []}
          style={undefined}>
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "padding": 20,
              }
            }>
            <View
              size={25}
              style={
                Object {
                  "alignItems": "center",
                  "flex": 25,
                  "flexDirection": "column",
                }
              }>
              <Image
                source={
                  Object {
                    "uri": "https://lh6.googleusercontent.com/-FuG2lZ4ub3o/AAAAAAAAAAI/AAAAAAAAA2c/FVy4JfdSKss/s128/photo.jpg",
                  }
                }
                style={
                  Object {
                    "borderRadius": 20,
                    "height": 40,
                    "width": 40,
                  }
                } />
            </View>
            <View
              size={75}
              style={
                Object {
                  "flex": 75,
                  "flexDirection": "column",
                }
              }>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#5e9fa1",
                    "fontSize": 16,
                    "fontWeight": "600",
                  }
                }>
                Tracy Loring
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#786048",
                  }
                }>
                Rating: 
                5
                 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={10}
                style={
                  Object {
                    "color": "#786048",
                    "fontSize": 14,
                    "lineHeight": 20,
                    "marginTop": 8,
                    "textAlign": "left",
                  }
                }>
                This is a gorgeous trail with lots of spots to hike down to the river. The trailhead is at the top with not a huge amount of parking. 
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "backgroundColor": "#e3e0d7",
                "height": 1,
              }
            } />
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "padding": 20,
              }
            }>
            <View
              size={25}
              style={
                Object {
                  "alignItems": "center",
                  "flex": 25,
                  "flexDirection": "column",
                }
              }>
              <Image
                source={
                  Object {
                    "uri": "https://lh6.googleusercontent.com/--5g7p0LNWTY/AAAAAAAAAAI/AAAAAAAAATs/WgPL6no7yiQ/s128/photo.jpg",
                  }
                }
                style={
                  Object {
                    "borderRadius": 20,
                    "height": 40,
                    "width": 40,
                  }
                } />
            </View>
            <View
              size={75}
              style={
                Object {
                  "flex": 75,
                  "flexDirection": "column",
                }
              }>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#5e9fa1",
                    "fontSize": 16,
                    "fontWeight": "600",
                  }
                }>
                Justin P
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#786048",
                  }
                }>
                Rating: 
                5
                 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={10}
                style={
                  Object {
                    "color": "#786048",
                    "fontSize": 14,
                    "lineHeight": 20,
                    "marginTop": 8,
                    "textAlign": "left",
                  }
                }>
                Kickoff point for beautiful hikes along the Salmon River. Bring a raincoat during the spring. Dog friendly.
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "backgroundColor": "#e3e0d7",
                "height": 1,
              }
            } />
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "padding": 20,
              }
            }>
            <View
              size={25}
              style={
                Object {
                  "alignItems": "center",
                  "flex": 25,
                  "flexDirection": "column",
                }
              }>
              <Image
                source={
                  Object {
                    "uri": "https://lh3.googleusercontent.com/-1aRrzAAWF2Y/AAAAAAAAAAI/AAAAAAAAHAA/fUGaaez9djA/s128/photo.jpg",
                  }
                }
                style={
                  Object {
                    "borderRadius": 20,
                    "height": 40,
                    "width": 40,
                  }
                } />
            </View>
            <View
              size={75}
              style={
                Object {
                  "flex": 75,
                  "flexDirection": "column",
                }
              }>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#5e9fa1",
                    "fontSize": 16,
                    "fontWeight": "600",
                  }
                }>
                Ricardo Hern√°ndez
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#786048",
                  }
                }>
                Rating: 
                4
                 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={10}
                style={
                  Object {
                    "color": "#786048",
                    "fontSize": 14,
                    "lineHeight": 20,
                    "marginTop": 8,
                    "textAlign": "left",
                  }
                }>
                Beautiful trail. Pack lunch and enjoy it by the river.
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "backgroundColor": "#e3e0d7",
                "height": 1,
              }
            } />
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "padding": 20,
              }
            }>
            <View
              size={25}
              style={
                Object {
                  "alignItems": "center",
                  "flex": 25,
                  "flexDirection": "column",
                }
              }>
              <Image
                source={
                  Object {
                    "uri": "https://lh4.googleusercontent.com/-WGG1XLtiKNc/AAAAAAAAAAI/AAAAAAAAAyA/Nmpt0PXl8dQ/s128/photo.jpg",
                  }
                }
                style={
                  Object {
                    "borderRadius": 20,
                    "height": 40,
                    "width": 40,
                  }
                } />
            </View>
            <View
              size={75}
              style={
                Object {
                  "flex": 75,
                  "flexDirection": "column",
                }
              }>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#5e9fa1",
                    "fontSize": 16,
                    "fontWeight": "600",
                  }
                }>
                Andrew Fry
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#786048",
                  }
                }>
                Rating: 
                5
                 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={10}
                style={
                  Object {
                    "color": "#786048",
                    "fontSize": 14,
                    "lineHeight": 20,
                    "marginTop": 8,
                    "textAlign": "left",
                  }
                }>
                Love this trail!! Beautiful river, great fishing.
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "backgroundColor": "#e3e0d7",
                "height": 1,
              }
            } />
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "padding": 20,
              }
            }>
            <View
              size={25}
              style={
                Object {
                  "alignItems": "center",
                  "flex": 25,
                  "flexDirection": "column",
                }
              }>
              <Image
                source={
                  Object {
                    "uri": "https://lh3.googleusercontent.com/-GW8RNh1QUFk/AAAAAAAAAAI/AAAAAAAAAc0/0rOOVA9lHBg/s128/photo.jpg",
                  }
                }
                style={
                  Object {
                    "borderRadius": 20,
                    "height": 40,
                    "width": 40,
                  }
                } />
            </View>
            <View
              size={75}
              style={
                Object {
                  "flex": 75,
                  "flexDirection": "column",
                }
              }>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#5e9fa1",
                    "fontSize": 16,
                    "fontWeight": "600",
                  }
                }>
                Aaron Taylor
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#786048",
                  }
                }>
                Rating: 
                4
                 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                numberOfLines={10}
                style={
                  Object {
                    "color": "#786048",
                    "fontSize": 14,
                    "lineHeight": 20,
                    "marginTop": 8,
                    "textAlign": "left",
                  }
                }>
                Beautiful cool hike next to the river
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "backgroundColor": "#e3e0d7",
                "height": 1,
              }
            } />
        </ScrollView>
      </View>
      <View
        size={40}
        style={
          Object {
            "flex": 40,
            "flexDirection": "column",
          }
        }>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
            }
          }>
          <View
            size={60}
            style={
              Object {
                "flex": 60,
                "flexDirection": "column",
                "padding": 20,
              }
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#2f5e4e",
                  "fontSize": 16,
                  "fontWeight": "600",
                  "paddingBottom": 10,
                }
              }>
              Old Salmon River Trailhead
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#786048",
                }
              }>
              Welches
              OR 97055
              USA
            </Text>
          </View>
          <View
            size={40}
            style={
              Object {
                "alignItems": "center",
                "flex": 40,
                "flexDirection": "column",
                "padding": 10,
              }
            }>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Object {
                    "alignItems": "center",
                    "padding": 20,
                  },
                ]
              }
              testID={undefined}>
              <Text
                accessible={true}
                allowFontScaling={false}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#e56452",
                      "fontSize": 30,
                    },
                    undefined,
                    Object {
                      "fontFamily": "FontAwesome",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                  ]
                }>
                ÔÄÜ
              </Text>
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Object {
                    "alignItems": "center",
                    "padding": 20,
                  },
                ]
              }
              testID={undefined}>
              <Text
                accessible={true}
                allowFontScaling={false}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#5e9fa1",
                      "fontSize": 25,
                    },
                    undefined,
                    Object {
                      "fontFamily": "FontAwesome",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                  ]
                }>
                Ôâπ
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#e3e0d7",
          "height": 1,
        }
      } />
  </View>
</View>
`;
